server/plugin.go

```diff
@@ -270,6 +274,19 @@ func (p *Plugin) getGitlabUserInfoByMattermostID(userID string) (*gitlab.UserInf
 	}
 
 	userInfo.Token.AccessToken = unencryptedToken
+	newToken, err := p.checkAndRefreshToken(userInfo.Token)
+	if err != nil {
+		return nil, &APIErrorResponse{ID: "", Message: err.Error(), StatusCode: http.StatusInternalServerError}
+	}
+
+	if newToken != nil {
+		userInfo.Token = newToken
+		unencryptedToken := newToken.AccessToken // needed because the storeGitlabUserInfo method changes its value to an encrypted value
```

Comment:
It is already defined above.
```suggestion
		unencryptedToken = newToken.AccessToken // needed because the storeGitlabUserInfo method changes its value to an encrypted value
```